spec/unit/graph/simple_graph_spec.rb

```diff
@@ -681,16 +686,20 @@ def graph_to_yaml(graph, which_format)
       end
 
       it "should be able to serialize a graph where the vertices contain backreferences to the graph (#{which_format} format)" do
+        pending('Make SimpleGraph a known to the Puppet Type System')
```

Comment:
Why does SimpleGraph need to be known to the type system? For example, the type system doesn't know about Reports, but `Report#to_data_hash` works as expected.